docs/user_guide/getting_started/minimal_script.rst

```diff
@@ -0,0 +1,115 @@
+.. currentmodule:: globus_sdk
+
+.. _minimal_script_tutorial:
+
+Creating A Minimal Script
+=========================
+
+This is a basic tutorial in the use of the Globus SDK.
+
+You can jump right in by using the ``CLIENT_ID`` seen in the example code blocks below!
+That is the ID of the tutorial client, which lets you get started quickly and easily.
+
+When you are ready to create your own application, follow :ref:`the tutorial on
+registering a Native App <tutorial_register_app>` and use its ``CLIENT_ID`` in
```

Comment:
I would inherit the name of the document as the link text rather than writing a a custom one.
It flows nicely enough in this sentence and helps codify in the users mind the name as a thing they can go back to.